packages/runtime/src/next/client.ts

```diff
@@ -201,23 +230,51 @@ export class Makeswift {
     return response
   }
 
-  async getPages({
-    siteVersion = MakeswiftSiteVersion.Live,
-  }: {
-    siteVersion?: MakeswiftSiteVersion
-  } = {}): Promise<MakeswiftPage[]> {
-    const response = await this.fetch(`v3/pages`, siteVersion)
+  private getPagesInternal = async (
```

Comment:
How come we changed this from a method to a field with an arrow function?

Reply:
This is an internal method that we are passing to `toIterablePaginationResult` wrapper. Defining it as a class field auto-binds `this`.

Reply:
Got it, I didn't notice we were relying on `this.fetch`. We could pay the price of instantiation at function call time by doing something like this:

```ts
  private async getPagesInternal() {
    //
  }

  getPages(...args) {
    return toIterablePaginationResult(this.getPagesInternal.bind(this))(...argrs)
  }
```

This makes it so the instances share all methods... but again, this is a super nit so what we have right now is perfectly ok.